client/src/components/linked-story/linked-story.js

```diff
@@ -29,8 +29,35 @@ const getClassName = story =>
     ? `${styles.icon} ${styles.story_icon_completed}`
     : styles.icon;
 
+const getInitials = name => {
```

Comment:
I misread this at first because `getInitials` is SO dangerously close to the word `genitals` 😬

Reply:
also, this would definitely blow up if passed `null`